nuxt-app/pages/pie-assistive-text.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+    <h1>Nuxt - PIE Assistive Text</h1>
+
+    <pie-link href="/">Home Page</pie-link>
+
```

Comment:
I expect you could potentially leverage a layout component to reusable this rather than adding the same markup everywhere